src/Campaigns/Blocks/CampaignDonationsBlock/render.php

```diff
@@ -0,0 +1,251 @@
+<?php
+
+namespace Give\Campaigns\Blocks\CampaignDonationsBlock;
+
+use Give\Campaigns\CampaignDonationQuery;
+use Give\Campaigns\Models\Campaign;
+use Give\Campaigns\Repositories\CampaignRepository;
+use Give\DonationForms\Blocks\DonationFormBlock\Controllers\BlockRenderController;
+use Give\Donations\ValueObjects\DonationMetaKeys;
+use Give\Framework\Support\ValueObjects\Money;
+
+/**
+ * @unreleased
+ */
+if ( ! class_exists(CampaignDonationsBlockRenderer::class)) {
```

Comment:
Why do we have to check if the class exists?

Reply:
Honestly, I didn't go much further on that but when in the Page Editor, I was receiving a fatal error from the block renderer saying that class was being registered multiple times.

Reply:
This is because the `CampaignDonationsBlockRenderer` is defined in the render.php file, and this file is included each time you include this block. The solution would be to move the class definition outside the render.php

Reply:
Refactored on 9392c1085bb153535a92e0fae8d0c3ce9b478e47